webpack/webpack.config.base.js

```diff
@@ -25,14 +25,16 @@ const getSegmentKey = () => {
 
 const getBranch = () => {
   try {
-    return execSync(`git rev-parse --abbrev-ref HEAD`, { encoding: 'utf8' }).trim();
+    const branch = execSync(`git rev-parse --abbrev-ref HEAD`, { encoding: 'utf8' }).trim();
```

Comment:
this command is what errors, we need to remove any `git` related commands

Reply:
Yeah, if it does error it returns null, and then if `branch` is null then the `commit` is null. It also sets these both to null if git doesn't error and the branch is `main` - so that the production build matches.